### DISCUSSION TITLE
Monogame and Visual Studio 2019 Community

### DISCUSSION THREAD

**Discussion by Kubrix** [MonoGame/MonoGame#8198] [General]
Does Visual Studio 2019 Community support Monogame?
Windows 7 

Downloaded https://github.com/MonoGame/MonoGame/releases/download/v3.7.1/MonoGameSetup.exe]MonoGameSetup.exe
(Link  sources
https://community.monogame.net/t/how-i-can-use-monogame-in-visual-studio-2017/17770 -> https://community.monogame.net/t/monogame-3-7-1/11173)

But installer does not have the "Community Templates" option.

**Comment by ThomasFOG** [comment]
3.7 is an old release and MonoGame is longer distributed via an installer.

Redirecting you to the [getting started page](https://monogame.net/articles/getting_started/index.html).

Visual Studio 2022 is required. MonoGame requires .NET 6, which is not supported by VS2019.

**Comment by Kubrix** [comment]
Okay
